tensorrt_llm/bench/benchmark/low_latency.py

```diff
@@ -178,6 +179,13 @@ def latency_command(
     """Run a latency test on a TRT-LLM engine."""
 
     logger.info("Preparing to run latency benchmark...")
+
+    if params.get("backend") is None:
```

Comment:
I would set the default to PyTorch if it's meant to be the default. I don't think setting to none is required.